src/transaction_builder.js

```diff
@@ -176,9 +191,6 @@ TransactionBuilder.fromTransaction = function (transaction, network) {
       throw new Error('coinbase inputs not supported')
     }
 
-    // Ignore empty scripts
-    if (txIn.script.length === 0) return {}
-
```

Comment:
moved this into `extractInput` so in the future it can first check if it's segwit